mmdet/apis/ote/apis/detection/nncf_task.py

```diff
@@ -65,6 +66,7 @@ def __init__(self, task_environment: TaskEnvironment):
         self._val_dataloader = None
         self._compression_ctrl = None
         self._nncf_preset = "nncf_quantization"
+        self._optimization_type = ModelOptimizationType.NNCF
```

Comment:
What should be the optimization type for the model, which is exported out of the NNCF task?

Reply:
After optimize and export optimization_type should be NNCF

Reply:
Hm...